contracts/Vault.vy

```diff
@@ -1563,13 +1546,17 @@ def _assessFees(strategy: address, gain: uint256):
             gain * self.strategies[strategy].performanceFee
         ) / MAX_BPS
         # NOTE: Unlikely to throw unless strategy reports >1e72 harvest profit
-        governance_fee += gain * self.performanceFee / MAX_BPS
+        performance_fee = gain * self.performanceFee / MAX_BPS
 
     # NOTE: This must be called prior to taking new collateral,
     #       or the calculation will be wrong!
     # NOTE: This must be done at the same time, to ensure the relative
     #       ratio of governance_fee : strategist_fee is kept intact
-    total_fee: uint256 = governance_fee + strategist_fee
+    total_fee: uint256 = performance_fee + strategist_fee + management_fee
+    # ensure total_fee is not more than gain
+    if total_fee > gain:
+        total_fee = gain
+        management_fee = total_fee - performance_fee - strategist_fee
```

Comment:
Same computed value, just makes it more clear what's going on here.
```suggestion
        management_fee = gain - performance_fee - strategist_fee
```

Also, this could be a DoS vuln on `report` if `performance_fee + strategist_fee > gain`, which shouldn't really happen in practice but is possible if `performanceFee + StrategyParams.performanceFee > 10_000 # BPS`